There is still a small formatting issue:

```
./src/vector/v.kriging/main.c:385:25: error: code should be clang-formatted [-Wclang-format-violations]
    if (xD.phase == 0) {
                        ^
Failed on file: ./src/vector/v.kriging/main.c

Error: Process completed with exit code 1.
```

Let me recommend to use **pre-commit**:
https://github.com/OSGeo/grass/blob/main/doc/development/submitting/submitting.md#use-pre-commit
which IMHO catches such formatting issues (black for Python, markdown, C, etc).

--- a/src/vector/v.kriging/main.c
+++ b/src/vector/v.kriging/main.c
@@ -383,7 +383,7 @@ int main(int argc, char *argv[])
 
     // Open report file if desired
     if (xD.phase == 0) {
-	report->name = opt.report->answer;
+        report->name = opt.report->answer;
 
         if (report->name) {
             // initial phase: check if the file exists

